custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1155,6 +1155,18 @@
           "type": "property_name",
           "property_name": "num_pnc_visits"
         }
+     },
+     {
+        "transform": {},
+        "datatype": "small_integer",
+        "type": "expression",
+        "is_nullable": true,
+        "column_id": "num_anc_complete",
+        "expression": {
+          "datatype": null,
+          "type": "property_name",
+          "property_name": "num_anc_complete"
```

Comment:
What's the difference between this and `num_anc_complete` from the pregancy tasks case? https://github.com/dimagi/commcare-hq/blob/2e735040599b3cc97c2526bbda2ba94e00c766a5/custom/icds_reports/ucr/data_sources/dashboard/pregnant_tasks.json#L89

Reply:
No clue. I just chose the one from the spec.